js/mainTarea.js

```diff
@@ -0,0 +1,47 @@
+const $nombre = document.formulario.nombre.value;
+function validarNombre(nombre) {
+	if (nombre.length >= 50) {
+		return "Mas de 50 caracteres";
+	}
+	if (nombre.length === 0) {
+		return "No escribiste nada";
+	} else {
+		return "";
+	}
+}
+
+const $ciudad = document.formulario.ciudad.value;
+
+function validarCiudad(ciudad) {
+	if (ciudad.length === 0) {
+		return "No se eligio ninguna ciudad";
+	} else {
+		return "";
+	}
+}
+const $comportamiento = document.formulario.comportamiento.value;
```

Comment:
Acá estás agarrando el .value de un elemento, pero lo estás guardando en una variable `$comportamiento` que me da a entender que es un elemento HTML.

Reply:
Got it. Se me hizo costumbre mandarle el $ a todo lo que sea DOM.